build.sbt

```diff
@@ -145,22 +145,27 @@ lazy val server = (project in file("server")) enablePlugins(JavaAppPackaging) se
       ExclusionRule("com.fasterxml.jackson.core"),
       ExclusionRule("com.fasterxml.jackson.module")
     ),
-    "org.apache.hadoop" % "hadoop-common" % "2.10.1" excludeAll(
+    "org.apache.hadoop" % "hadoop-common" % "3.3.4" excludeAll(
       ExclusionRule("com.fasterxml.jackson.core"),
       ExclusionRule("com.fasterxml.jackson.module"),
       ExclusionRule("com.google.guava", "guava")
     ),
-    "org.apache.hadoop" % "hadoop-client" % "2.10.1" excludeAll(
+    "org.apache.hadoop" % "hadoop-client" % "3.3.4" excludeAll(
       ExclusionRule("com.fasterxml.jackson.core"),
       ExclusionRule("com.fasterxml.jackson.module"),
       ExclusionRule("com.google.guava", "guava")
     ),
-    "org.apache.parquet" % "parquet-hadoop" % "1.10.1" excludeAll(
+    "org.apache.parquet" % "parquet-hadoop" % "1.12.3" excludeAll(
       ExclusionRule("com.fasterxml.jackson.core"),
       ExclusionRule("com.fasterxml.jackson.module"),
       ExclusionRule("com.google.guava", "guava")
     ),
-    "io.delta" %% "delta-standalone" % "0.5.0" excludeAll(
+    "io.delta" % "delta-kernel-defaults" % "3.2.0" excludeAll(
```

Comment:
This one is also a duplicate of the import below?

Reply:
fixed